modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -370,4 +370,48 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 			/>
 		</div>
 	</div>
-</c:if>
\ No newline at end of file
+</c:if>
+
+<aui:script require="frontend-js-web/liferay/delegate/delegate.es as delegateModule">
+	var delegate = delegateModule.default;
+
+	var submitButtonClickEventHandler = delegate(
+		document.querySelector('#<%= ddmFormDisplayContext.getContainerId() %>'),
+		'click',
+		'.lfr-ddm-form-submit',
+		(event) => {
+			var sessionStatus = Liferay.Session.get('sessionState');
```

Comment:
@carolmariaabb, shouldn't we call this variable as `sessionState` to keep consistency?

Besides, is there any special reason for not setting the type of this variable to `let` instead of `var`?

Reply:
No reason, @natocesarrego. I will change it to `let`.

Thanks!